AudioConfigTeensy4_10bitPwm.h

```diff
@@ -0,0 +1,17 @@
+#ifndef AUDIOCONFIGTEENSY4_10BITPWM_H
```

Comment:
I have no idea what other output options would be "natural" on teensy 4, and how similar these would be to set up. But just as a point to consider. Could this be too specialized? Would it be better to use plain AudioConfigTeensy4.h, instead?